data/language/de-DE.yml

```diff
@@ -118,7 +118,7 @@ strings:
   104: Spiel speichern
   105: Szenario-Editor beenden
   106: Spiel beenden
-  107: Screenshot{INT16}.BMP
+  107: Screenshot{INT16}.png
```

Comment:
Just put no file extension in the translation files. We'll always save as png anyways, and if we ever add something like webp-support, we don't need to change the translations.